Sources/PointFree/Episode/Episode.swift

```diff
@@ -95,10 +95,12 @@ public struct Episode {
   }
 
   public struct Exercise {
-    public private(set) var body: String
+    public var problem: String
```

Comment:
renamed `body` to `problem`. caused a lot of churn, but think it makes more sense. I've got the changes in the transcripts directory in a branch: https://github.com/mbrandonw/episode-transcripts/compare/model-update?expand=1